src/views/product-tutorials-view/components/product-view-content/components/branded-callout/index.tsx

```diff
@@ -3,26 +3,30 @@ import s from './branded-callout.module.css'
 
 function BrandedCallout({
   heading,
+  headingSlug,
```

Comment:
All top-level `product-view-content` blocks now support a `headingSlug` prop, which keeps them in sync with the `#heading-slug` links we're rendering in the `Sidecar`.